src/views/TwabRewards/hooks/useAccountsPromotions.ts

```diff
@@ -0,0 +1,62 @@
+import gql from 'graphql-tag'
+import { useQuery } from 'react-query'
+import { NO_REFETCH } from '@pooltogether/hooks/dist/constants'
+
+import { useTwabRewardsSubgraphClient } from '@twabRewards/hooks/useTwabRewardsSubgraphClient'
+
+/**
+ *
+ * @param chainId
+ * @param account
+ * @returns Promotion[]
+ */
+export const useAccountsPromotions = (chainId: number, account: string) => {
```

Comment:
nit: I prefer `useUsersPromotions` and `usersAddress: string` for consistency. Not a big deal though.

Reply:
Ah I see, `account` comes from the subgraph